editor/store/effects.js

```diff
@@ -49,9 +49,7 @@ import {
 	getCurrentPostType,
 	getEditedPostContent,
 	getPostEdits,
-	isCurrentPostPublished,
 	isEditedPostDirty,
-	isEditedPostNew,
```

Comment:
Why has this been removed? This is causing an error because the `AUTOSAVE` effect attempts to call it:

>Uncaught ReferenceError: isEditedPostNew is not defined

Reply:
that usage is now removed